prime-dotnet-webapi/ViewModels/Agreements/_AgreementMappingProfile.cs

```diff
@@ -0,0 +1,23 @@
+using AutoMapper;
+
+using Prime.Models;
+
+namespace Prime.ViewModels.Profiles
+{
+    public class AgreementMappingProfile : Profile
```

Comment:
any benefit using underscore prefix for the class filename like `_AgreementMappingProfile.cs`?

Reply:
Putting the underscore makes the profile bubble up to the top of the file list, making it easier to find. 
![image](https://user-images.githubusercontent.com/39168456/132767626-9533e14b-eb01-4b14-8e3b-5decec3fad3d.png)

vs

![image](https://user-images.githubusercontent.com/39168456/132767659-70709163-1a19-4092-877c-56bc789da8dd.png)